### PR TITLE
Fortnox

### PR DESCRIPTION
**Pull request by rippo**

### CODE REVIEW

File: WebcrmConnector/RemoteCalls/BaseApi.cs [hunk 1]
```diff
@@ -5,23 +5,27 @@ namespace Webcrm.Integrations.WebcrmConnector.RemoteCalls
 {
```
Comment (janaagaard75) [inline WebcrmConnector/RemoteCalls/BaseApi.cs]: Delete this file.

**Comment by rippo** [comment]
This is a test comment

**Comment by janaagaard75** [comment]
Another test comment.

File: WebcrmConnector/RemoteCalls/BaseApi.cs (hunk 1 above)
Comment (rippo) [reply WebcrmConnector/RemoteCalls/BaseApi.cs]: File deleted

**Pull request merged**